api/cases/case.js

```diff
@@ -43,17 +51,19 @@ module.exports = (req, res, next) => {
         }
     };
 
-    getCaseWithEventsAndQuestions(caseId, userId, options)
+    getCaseWithEventsAndQuestions(caseId, userId, options, casetype, jurisdiction)
         .then(([caseData, events, questions]) => {
             caseData.questions = questions;
```

Comment:
wouldn't this cause an error if your judristion wasn't `sscs`? or does it just return an empty array or somthing?

Reply:
It will return null for questions